spec/betterc.dd

```diff
@@ -84,6 +84,7 @@ $(H2 $(LNAME2 retained, Retained Features))
     $(LI `switch` with strings)
     $(LI `final switch`)
     $(LI `unittest`)
+    $(DDSUBLINK spec/interfaceToC, calling_printf, `printf` format validation)
```

Comment:
Needs to be wrapped in `$(LI ... )` as it's part of `$(OL ..)` (ordered html list). I'll fix it myself.

Reply:
(Done: https://github.com/dlang/dlang.org/compare/175ea2ce01a84fb99d28ccb2d78b70196e1e949f..09364b2ea0a09e7d312682b473cabd4863af49f7)